tests/testthat/test-rxode-options.R

```diff
@@ -0,0 +1,34 @@
+test_that("rxCreateCache", {
```

Comment:
@mattfidler, If caches are used during model testing or otherwise, this could cause models to recompile. These tests may be better to skip most of the time.

Reply:
While I am all for testing, this would definitely break parallelism (if ever implemented)